common/src/main/java/com/wynntils/features/debug/PacketDebuggerFeature.java

```diff
@@ -40,7 +40,7 @@ public MutableComponent getNameComponent() {
             ClientboundSetEquipmentPacket.class,
             // Chunks
             ClientboundForgetLevelChunkPacket.class,
-            ClientboundLevelChunkPacket.class,
+            // ClientboundLevelChunkPacketData.class,
```

Comment:
Why is this commented out?

Reply:
This class does not exist in 1.18. As its a debug feature I did not spend much time looking into it.

Reply:
Ah, right, I missed the context of this list. Just remove the class completely. (I added to the list the packets that tended to spam the packet stream. If it has been replaced by another, it will soon become apparent that another class needs to be added.)